File: Reverse a String.java
// logic for reverse a string
class Reverse
{
    // Complete the function
    // str: input strin

    public static String reverseWord(String str)
    {   String rev="";
       for(int i =str.length()-1;i>=0;i--)
       {
           rev+=str.charAt(i);
       }
       return rev;
    }
}

File: Reverse a String.cpp
//https://practice.geeksforgeeks.org/problems/reverse-a-string/1#

string reverseWord(string str){
     char s=0,e=str.size()-1,m; 
        for(int i=0;i<e;i++,e--)
        {
            m=str[i];
            str[i]=str[e];
            str[e]=m; 
        }
       return str;
}